.github/workflows/kata-runtime-classes-sync.yaml

```diff
@@ -0,0 +1,36 @@
+on:
```

Comment:
When you said in the other comment we would keep the all-in-one file I immediately thought "It can get un-sync easily". So That's a good idea to create a checker for this, but I think that using an action is too much. I hear voices in my mind, they say "static-checker tool.. static-checker tooolllll".

Reply:
Hmm.  There's no way I'm going to touch the static-checker right now, at least not till it gets moved to main and maybe even refactored. :-)

For now, can we just live with the test we have? :-)

Reply:
I am okay with that. I just suggest to open an issue to track this as a possible improvement.